client/my-sites/email/email-non-domain-owner-message/index.tsx

```diff
@@ -107,9 +120,14 @@ export const EmailNonDomainOwnerMessage = ( props: EmailNonDomainOwnerMessagePro
 
 	return (
 		<>
-			<PromoCard className="email-non-domain-owner-message__non-owner-notice">
-				<p>{ reasonText }</p>
-			</PromoCard>
+			{ usePromoCard && (
+				<PromoCard className="email-non-domain-owner-message__non-owner-notice">
+					<p>{ reasonText }</p>
+				</PromoCard>
+			) }
+			{ ! usePromoCard && (
+				<p className={ 'email-non-domain-owner-message__non-owner-message' }>{ reasonText }</p>
+			) }
```

Comment:
That's only a suggestion but I find the following form syntax clear and shorter:

```suggestion
			{ usePromoCard ? (
				<PromoCard className="email-non-domain-owner-message__non-owner-notice">
					<p>{ reasonText }</p>
				</PromoCard>
			) : (
				<p className={ 'email-non-domain-owner-message__non-owner-message' }>{ reasonText }</p>
			) }
```